docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,238 @@
+:orphan:
+
+.. doctest-skip-all
+
+******************************
+Defining Observing Constraints
+******************************
+
+Contents
+========
+
+* :ref:`constraints-built_in_constraints`
+* :ref:`constraints-user_defined_constraints`
+
+.. _constraints-built_in_constraints:
+
+Built-In Constraints
+====================
+
+Frequently, we have a long list of targets that we want to observe, and we need
+to know which ones are observable given a set of constraints imposed on our
+observations by a wide range of limitations. For example, your telescope may
+only point over a limited range of altitudes, your targets are only useful
+in a range of airmasses, and they must be separated from the moon by some
+large angle. The `astroplan.constraints` module is here to help!
```

Comment:
The `astroplan.constraints` module is not a namespace that's part of the official astroplan API.
At the moment everything is exposed via the top-level `astroplan` namespace.

So either change this to not refer to `astroplan.constraints` (probably easier) or expose the `astroplan.constraints` namespace (and not put it's content in the top-level namespace) (also very reasonable, but not what we discussed / agreed so far).